host/tests/L3_TestCases/dsCompositeIn/dsCompositeInHelperClass.py

```diff
@@ -137,21 +101,14 @@ def testPrepareFunction(self):
             bool
         """
 
-        # Download the assets listed in test setup configuration file
-        self.testDownloadAssets()
-
         # Run Prerequisites listed in the test setup configuration file
         self.testRunPrerequisites()

```

Comment:
in the dsCompositeIn_L3_testSetup.yml
these is no execute still this is required ?
# Run commands as part of test prerequisites
        test = self.testSetup.get("assets").get("device").get(self.testName)
        cmds = test.get("execute")
        if cmds is not None:
            for cmd in cmds:
                self.writeCommands(cmd)

Reply:
Removed the testRunPrerequisite function